tests/unit/certGen.unitTest.ts

```diff
@@ -2093,6 +2094,462 @@ describe("cert-gen", () => {
         });
     });
 
+    context("CertGenService for ADR", () => {
+        context("when a passing test result for ADR is read from the queue", () => {
+            const event: any = lodash.cloneDeep(queueEventPass);
+            const testResult: any = JSON.parse(event.Records[1].body);
+            testResult.testTypes.testTypeId = "50";
+
+            context("and a payload is generated", () => {
+                context("and no signatures were found in the bucket", () => {
+                    it("should return an ADR_PASS payload without signature", () => {
+                        const expectedResult: any = {
```

Comment:
Can this be place in resources? It would reduce the lines of code and make the unit test more readable.

Reply:
fixed